ann_benchmarks/results.py

```diff
@@ -30,3 +30,22 @@ def load_results(dataset, count, definitions):
             f = h5py.File(fn)
             yield definition, f
             f.close()
+
+def _get_leaf_paths(path):
```

Comment:
think you can just use os.walk here and make this a few lines shorter (don't have to implement the recursion yourself)

Reply:
Done. Thanks for the suggestions.